tests/app.test.ts

```diff
@@ -1,8 +1,99 @@
 import request from "supertest";
-import  app from "../source/app";
+import router from '../routes/routes';
+import { Soldier } from "../repository/repo";
+const app = router; 
+beforeEach(async ()=>{ 
+    await Soldier.deleteMany();
```

Comment:
This has nothing to do with the app tests, either make sure your tests don't go down that far, or make a cleanup function in a separate spot that you can call from here